src/modules/edc-demo/components/contract-agreement-transfer-dialog/model/contract-agreement-datasink-form-builder.ts

```diff
@@ -0,0 +1,93 @@
+import {Injectable} from '@angular/core';
+import {FormBuilder, FormGroup, Validators} from '@angular/forms';
+import {switchDisabledControls} from '../../../utils/form-group-utils';
+import {jsonValidator} from '../../../validators/json-validator';
+import {urlValidator} from '../../../validators/url-validator';
+import {DataAddressType} from '../../data-address-type-select/data-address-type';
+import {
+  ContractAgreementDatasinkFormModel,
+  ContractAgreementDatasinkFormValue,
+} from './contract-agreement-datasink-form-model';
+import {HttpDatasinkAuthHeaderType} from './http-datasink-auth-header-type';
+import {HttpDatasinkHeaderFormModel} from './http-datasink-header-form-model';
+
+@Injectable()
+export class ContractAgreementDatasinkFormBuilder {
+  constructor(private formBuilder: FormBuilder) {}
+
+  buildFormGroup(): FormGroup<ContractAgreementDatasinkFormModel> {
+    const datasink: FormGroup<ContractAgreementDatasinkFormModel> =
+      this.formBuilder.nonNullable.group({
+        dataAddressType: 'Http' as DataAddressType,
+        dataDestination: ['', [Validators.required, jsonValidator]],
+        publisher: ['', urlValidator],
+        standardLicense: ['', urlValidator],
+        endpointDocumentation: ['', urlValidator],
```

Comment:
publisher, standardLicense and endpointDocumentation are not used in the transfer dialog and should not exist in the form model then